cage/cryptids/templates/index.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load static %}
+
+
+{% block title %}Cryptids{% endblock %}
+{% block content %}
+
+
+
+
+<ul>
+        {% for animals in animalss %}
```

Comment:
For your `for loop`, you could just do `for animal in animals`, reads a bit more easily :)

Reply:
Yes I was a little confused on this what it should be thanks